policyengine_us/parameters/gov/states/va/tax/income/deductions/standard.yaml

```diff
@@ -10,14 +10,21 @@ metadata:
     href: https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/
   - title: Virginia 2022 Form 760 Resident Individual Income Tax Booklet
     href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=18
+  - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=18
 
 JOINT:
   2022-01-01: 16_000
+  2021-01-01: 9_000
```

Comment:
we put the 2021 variable first 
2021-01-01: 
2022-01-01: 

all parameter file